src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+export default function booksReducer(state = [], action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.id);
+    default:
+      return state;
```

Comment:
- [ ] Since you are trying to filter the book by it's id, `book.id !== action.id` is not sufficient. I believe you should do something like this 👇 
```
book.id !== action.book.id
```

Reply:
thank you for the feedback, now it's done.

Reply:
You're welcome, @Juanse7793 👍  Please go ahead and request another review.